tests/cache/tests.py

```diff
@@ -1008,6 +1008,17 @@ def tearDown(self):
         super().tearDown()
         self.drop_table()
 
+    def test_num_queries_of_get_many(self):
+        cache.set_many({'a': 1, 'b': 2})
+        cache.set('expired', 'expired', 0.01)
+
+        with self.assertNumQueries(1):
+            self.assertEqual(cache.get_many(['a', 'b']), {'a': 1, 'b': 2})
+
+        time.sleep(0.02)
+        with self.assertNumQueries(2):
+            self.assertEqual(cache.get_many(['a', 'b', 'expired']), {'a': 1, 'b': 2})
+
```

Comment:
please insert tests on line 1029, after the `setUp`/`tearDown` helpers `create_table` and `drop_table`